commands/rank.js

```diff
@@ -0,0 +1,141 @@
+const Discord = require('discord.js');
+const Enmap = require("enmap");
+
+const ALLOWED_RANKS = [
+  "Iron1",
+  "Iron2",
+  "Iron3",
+  "Bronze1",
+  "Bronze2",
+  "Bronze3",
+  "Silver1",
+  "Silver2",
+  "Silver3",
+  "Gold1",
+  "Gold2",
+  "Gold3",
+  "Platinum1",
+  "Platinum2",
+  "Platinum3",
+  "Diamond1",
+  "Diamond2",
+  "Diamond3",
+  "Immortal1",
+  "Immortal2",
+  "Immortal3",
+  "Radiant"
+];
+
+exports.run = async (client, message, args, level) => {
+  client.ranks = new Enmap({name: "ranks"});
```

Comment:
Would it be better to have this connection as a const for this class? Is it expensive to re-init the map (backed by sqlite) everytime?

Reply:
can we do it even though it depends on client? or does it even depend on client? the demo did it this way but I'l look into it